# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/blkpg.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./include/linux/blkpg.h.orig	2004-11-18 16:07:34.000000000 +0100
+++ ./include/linux/blkpg.h	2005-04-06 22:17:00.257102232 +0200
@@ -1,6 +1,8 @@
 #ifndef _LINUX_BLKPG_H
 #define _LINUX_BLKPG_H
 
+#include <linux/compiler.h>
+
 /*
  * Partition table and disk geometry handling
  *
